tests/_test_mscolab/test_server.py

```diff
@@ -245,13 +246,19 @@ def test_get_change_content(self):
         with self.app.test_client() as test_client:
             operation, token = self._create_operation(test_client, self.userdata)
             fm, user = self._save_content(operation, self.userdata)
+            # we need to wait to get an updated created_at
+            time.sleep(1)
             fm.save_file(operation.id, "content2", user)
             all_changes = fm.get_all_changes(operation.id, user)
+            # the newest change is on index 0, because it has a recent created_at time
             response = test_client.get('/get_change_content', data={"token": token,
                                                                     "ch_id": all_changes[1]["id"]})
             assert response.status_code == 200
             data = json.loads(response.data.decode('utf-8'))
             assert data == {'content': 'content1'}
+            assert all_changes[0]["id"] == 2
+            assert all_changes[0]["id"] > all_changes[1]["id"]
+            assert all_changes[0]["created_at"] > all_changes[1]["created_at"]
```

Comment:
This test now tests two things: the get_change_content endpoint and that two consecutive changes are in the right order. The test for the correct order should be moved to test_get_all_changes, similar to how it is in test_files_api.py.